### ISSUE TITLE
to be / om te wees / om te zijn

### ISSUE THREAD

**Issue by aaddrr** [apertium/apertium-afr-nld#4]
`<e><p><l>wees<s n="vbser"/></l><r>zijn<s n="vbser"/></r></p></e>`

Is there a way to accommodate the following construction: zij zijn <-> hulle is

**Comment by ftyers** [comment]
@aaddrr thanks for your comment, could you give some more details on the construction ? e.g. some example sentences -- we will see what we can do.

**Comment by aaddrr** [comment]
@ftyers Using content translation from Dutch Wikipedia to Afrikaans Wikipedia:

Ze zijn opvallend gekleurd -> Hulle is opvallend gekleurd (not _Hulle **wees** opvallend gekleurd_) (see https://nl.wikipedia.org/wiki/Rinofoor).

I can add more examples in separate comments as I encounter them.

**Comment by ftyers** [comment]
I've fixed this in 61ccc46, but I'm not sure how long this will take to get updated in ContentTranslation. @MarcRiera do you think you could monolingualise this pair, it seems it is being used so would make a good candidate. If there are a lot of testvoc errors I'm happy to participate in fixing them.

**Comment by MarcRiera** [comment]
I began work to upgrade this pair three days ago (see #6). The amount of testvoc errors is rather small, so the upgraded pair should be ready at the end of this week.

**Comment by ftyers** [comment]
@MarcRiera that's great thanks!

**Comment by jcwf** [comment]
Hi all, Thanks for your efforts on Dutch->Afrikaans.  I am totally new to github, have no intention to learn another programming language (I am 67).  I am Dutch but write mostly for Afrikaans wikipedia these days and I am trying to improve my Afrikaans, which is a struggle.  I have just used the translation tool on a small Dutch article and must admit that the result is pretty useless and would be scrapped at af.wiki for good reasons: we have few people there and drown in bad language 'contributions' and 'translations'. So, for this tool to become useful a lot must happen. Would it be useful if I report some items?

For example the Dutch word "het"  usually translates into "die" the definite article, e.g. "het land" becomes "die land"  (the land).  But not if it is used as an impersonal pronoun "Het is"  becomes "dit is" (it is) And this goes for all combinations with verbs: "het wordt"-> "dit word", "het heeft"-> "dit het"

**Comment by ftyers** [comment]
@jcwf yes... the tool was originally developed for Afrikaans → Dutch and the Dutch → Afrikaans direction hasn't really been developed much at all.

We would really benefit from bug reports, so if you find stuff please feel free to report it. The ideal thing would be to open up a separate issue for each thing you find. Although if you find things that are related feel free to put them in the same issue. 

I can't guarantee that the changes will be available quickly, but we do take note of all suggestions and are working on getting a faster turnaround. Having more bugs to fix will aid that process. :)

**Comment by jcwf** [comment]
@ftyers Thanks Francis,

Does this apply to individual vocabulary? E.g.  nl: nestplaats was mistranslated. It is af: nesplek

**Comment by ftyers** [comment]
@jcwf yep, please submit individual issues. Although with vocabulary if you find up to 10 problems, you can submit them as a single issue.
